projects/1.19/assets/chat-sounds/chatsounds/lang/zh_cn.json

```
@@ -0,0 +1,33 @@
+{
+    "text.autoconfig.chatsounds.title": "聊天音效",
+    "text.autoconfig.chatsounds.option.join": "上线信息",
```

```suggestion
    "text.autoconfig.chatsounds.option.join": "加入消息",
```
原版`%s joined the game`：`%s加入了游戏`